compiler/one-cmds/how-to-use-one-commands.txt

```diff
@@ -20,6 +20,16 @@ Compilation flow for NPU
 4) one-codegen will compile to binary codes.
 
 
+Prerequisite
+------------
+
+Before you use one commands, please prepare a python environment with TensorFlow installed.
+You can use `one-prepare-venv`(ubuntu) script where it automatically installs python3 virtual
+environment with Tensorflow-cpu.
+
+If you have prepared Python as a virtual environment, don't forget to activate it.
```

Comment:
Previouslly we didn't let users to activate but the shell script did it. Now we use python script and this makes activation not possible (or we don't know how-to).
What I could think of right now...
- add a script to activate and deactivate
- let one-import-tf as-is using bash
- ...

Reply:
@seanshpark I used this magic:)
```bash
#!/usr/bin/env bash
''''export SCRIPT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"                  # '''
''''export PY_PATH=${SCRIPT_PATH}/venv/bin/python                                       # '''
''''test -f ${PY_PATH} && exec ${PY_PATH} "$0" "$@"                                     # '''
''''echo "Error: Virtual environment not found. Please run 'one-prepare-venv' command." # '''
```
First, it will be run as a bash script. And then, check if python path exists. If it does, execute it. Triple quote will be comment then.

Reply:
This is very interesting!

Reply:
@seanshpark Actually, there is pros and cons.
`one-import-tf.py` + `one-import` = increase the number of files
`one-import-tf`(bash + python as above) = need manual format check and no ide formatting